src/Umbraco.Web/umbraco.presentation/umbraco/users/EditUser.aspx.cs

```diff
@@ -631,5 +638,13 @@ private void UpdateMembershipProvider(MembershipUser membershipUser)
         /// To modify move field declaration from designer file to code-behind file.
         /// </remarks>
         protected TabView UserTabs;
+        
+        internal BackOfficeUserManager<BackOfficeIdentityUser> GetBackofficeUserManager()

```

Comment:
Do we need to put this logic into these old classes?

Reply:
Yes :(
Fortunately this class is deleted in 7.7.0 :-)